diagrams.md

```diff
@@ -0,0 +1,548 @@
+        String provisioning_failure_cause
+        DateTime provisioning_request_date
+    }
+    TASK-ORDER {
+        GUID sys_id PK
+        Reference portfolio FK
+        Reference acquisition_package FK
+        Reference funding_plan FK
+        Reference funding_request FK
+        List clins FK
+        String task_order_number
+        Choice task_order_status
+        Date pop_start_date
+        Date pop_end_date
+        Currency funds_total
+        Currency funds_obligated
+        Choice incrementally_funded
+    }
+    CLIN {
+        GUID sys_id PK
+        Reference task_order FK
+        String clin_number
+        Choice clin_status
+        Choice idiq_clin
+        Date pop_start_date
+        Date pop_end_date
+        Currency funds_total
+        Currency funds_obligated
+    }
+    CLOUD-SERVICE-PROVIDER {    
+        GUID sys_id PK
+        String name
+        Choice network
+        String uri "change to URL type?"
+        URL pricing_calculator_uri
+    }
+    COSTS {
+        GUID sys_id PK
+        Reference csp FK
+        Reference portfolio FK
+        Reference organization FK
+        Reference agency FK
+        String task_order_number
+        String clin
+        Boolean is_actual "actual/forecast"
+        Currency value
+        Date year_month "ignore day"
+    }
+    AGENCY {
+        GUID sys_id PK
+        Integer css_id "Contract Support System"
+        String label
+        String title
+        String acronym
+    }
+    OPERATOR {
+        GUID sys_id PK
+        Reference portfolio FK
+        String dod_id
+        String email
+        Boolean needs_reset
+        Boolean provisioned
+        DateTime provisioning_request_date
+        DateTime provisioned_date
+        String provisioning_failure_cause
+        String added_by
+    }
+    PORTFOLIO ||--|{ TASK-ORDER : "funded by"
+    TASK-ORDER ||--|{ CLIN : "has multiple (from EDA)"
+    CLIN }|--|| TASK-ORDER : "part of"
+    PORTFOLIO ||--|{ COSTS : "has (from CSPs)"
+    PORTFOLIO ||--|{ SYS_USER : "portfolio managers and viewers"
+    PORTFOLIO ||--|{ OPERATOR : "pending operators"
+    COSTS }|--|| CLOUD-SERVICE-PROVIDER : "billed by"
+    COSTS }|--|{ TASK-ORDER : "billed against"
+    COSTS }|--|| ORGANIZATION : "incurred by"
+    COSTS }|--|| AGENCY : ""
+    ALERT }|--|| PORTFOLIO : ""
+    ALERT }|--|| TASK-ORDER : ""
+    ALERT }|--|| CLIN : "relates to"
+
+    %% Package
+    DITCO-CONTRACT-SPECIALIST {
+        GUID sys_id PK
+        Reference user FK "to sys_user"
+        String first_name
+        String last_name
+        Boolean is_ko
+        Integer css_id "Contract Support System"
+    }
+    IDIQ-CLIN {
+        GUID sys_id PK
+        Reference classification_level FK
+        Choice contract "JWCC"
+        Choice contract_type "FFP, T&M"
+        String description
+        String idiq_clin
+    }
+    ACQUISITION-PACKAGE {
+        GUID sys_id PK
+        Reference classification_level FK
```

Comment:
I'm just wondering is this needed here? From my understanding the `*_instances` reference the `classification_level`, so it doesn't seem like the `classification_level` is needed on the package.